tests/unit/handle-changes-test.js

```diff
@@ -0,0 +1,44 @@
+var simple = require('simple-mock')
+var test = require('tape').test
+
+var handleChanges = require('../../lib/helpers/handle-changes')
+
+test('handleChanges changes during bootstrapping', function (t) {
+  t.plan(2)
+
+  var change = {
+    doc: {
+      foo: 'bar'
+    },
+    id: 'id1234'
+  }
+  var changesSpy = {
+    on: simple.spy().callFn(function (changeEvent, callback) {
+      callback(change)
+    })
+  }
+
+  var state = {
+    db: {
+      allDocs: simple.spy().resolveWith({ rows: [] }),
+      changes: simple.spy().returnWith(changesSpy)
```

Comment:
I’m surprised that simple.spy() works here :) I think using `simple.mock()` would make more sense, a `.spy()` is usually used to count how often a method was called without altering its behaviour, while a `.mock()` can change the behavior, too. I guess both work in this case, but I’d prefer `simple.mock()`

Here is an example of what I mean

```js
var simple = require("simple-mock")
var foo = {
  bar: function () {console.log('BAR!') },
  baz: function () {console.log('BAZ!') }
}
simple.spy(foo, 'bar').callFn(() => {console.log('SPY!')})
simple.mock(foo, 'baz').callFn(() => {console.log('MOCK!')})
foo.bar() // logs BAR!
foo.baz() // logs MOCK!
```